packages/icons/package.json

```diff
@@ -22,6 +22,7 @@
     "./*": "./src/assets/*"
   },
   "scripts": {
+    "build": "tsc",
```

Comment:
Whats the purpose of this? The dist folder is not published in the npm package

Reply:
Done